bundles/FileExplorerBundle/public/js/file.js

```diff
@@ -11,11 +11,11 @@
  * @license    http://www.pimcore.org/license     GPLv3 and PCL
  */
 
-pimcore.registerNS("pimcore.settings.fileexplorer.file");
+pimcore.registerNS("pimcore.bundle.file_explorer.settings.file");
```

Comment:
Either move the js file under `/settings` folder or drop `settings` from namespace.